contrib/python-hypothesis/template.py

```diff
@@ -0,0 +1,67 @@
+pkgname = "python-hypothesis"
+pkgver = "6.96.2"
+pkgrel = 0
+build_wrksrc = "hypothesis-python"
+build_style = "python_pep517"
+hostmakedepends = [
+    "python-build",
+    "python-installer",
+    "python-setuptools",
+    "python-wheel",
+]
+checkdepends = [
+    "python-attrs",
+    "python-black",
+    "python-click",
+    "python-dateutil",
+    "python-pexpect",
+    "python-ptyprocess",
+    "python-pytest",
+    "python-pytz",
+    "python-pytest-xdist",
+    "python-sortedcontainers",
+    "python-typing_extensions",
+]
+depends = ["python-attrs", "python-sortedcontainers"]
+pkgdesc = "Easy to use library for property-based testing"
+maintainer = "Duncan Bellamy <dunk@denkimushi.com>"
+license = "MPL-2.0"
+url = "https://hypothesis.works/index.html"
+source = f"https://github.com/HypothesisWorks/hypothesis/archive/refs/tags/hypothesis-python-{pkgver}.tar.gz"
+sha256 = "80ee73f86f0c48f65a762e03db240e7d4a4f41d6c15d06f4425e3a0bf38c0403"
+# with check link gives the error:
+# python-hypothesis-6.93.0-r0: forbidden directory 'builddir'
+# after building
+options = ["!check"]
+
+
+def init_check(self):
+    self.env["PATH"] = f"{self.chroot_cwd}/.cbuild-checkenv/bin:/usr/bin"
+    self.make_check_args = [
+        "--ignore",
+        "tests/array_api",
```

Comment:
you can do --ignore=.. to halve the linecount

it can also be toplevel and not in init_check

and the PATH seems not needed??

Reply:
also put -n self.make_jobs in here since you pulled xdist, since these tests take forever

Reply:
hm, i guess for some reason the path is indeed missing from some of the tests

Reply:
okay

Reply:
you still gotta do --ignore=thing instead of making it two lines

Reply:
i assume `-n {self.make_jobs}` also does not work as a single argument

Reply:
eyah, could do `--numprocesses={}` instead

Reply:
okay changed it, do you mean change to `-n{self.make_jobs}`